src/UserSettingsPage.cpp

```diff
@@ -1134,6 +1166,12 @@ UserSettingsModel::data(const QModelIndex &index, int role) const
               "Change the appearance of user avatars in chats.\nOFF - square, ON - circle.");
         case UseIdenticon:
             return tr("Display an identicon instead of a letter when no avatar is set.");
+        case OpenImageExternal:
+            return tr("Click to open image with external program. \nSame as Right-Click>Open "
+                      "in external program");
```

Comment:
It might make sense to hint here, that files in encrypted rooms will be left unencrypted on disk if you enable that option and open such a file.

Reply:
Something like "Toggles the behavior of "Right-Click>Open with external program", but when tapping the image.
\nNote that files are left unencrypted on disk and must be manually deleted" is enough? maybe point where the cache folder is located?

Reply:
Well, the cache folder is different on every system, so you would need to use a variable for that then.